spec/game_team_spec.rb

```diff
@@ -17,9 +17,18 @@
          @game_teams << GameTeam.new(game_team_details)
       end
       @game_teams
+      @game_team_0 = @game_teams[0]
    end
 
    it 'exists' do
       expect(@game_teams[0]).to be_a GameTeam
+      expect(@game_team_0.game_id).to eq("2012030221")
```

Comment:
This way works great, and it's testing what we are looking for. I wonder if it's best practice to have an `it` block describing what we are testing and that is it 'has a game id'.